### PR TITLE
added table

### PR DESCRIPTION
**Pull request by sivakanili**
the PR adds a table

### CODE REVIEW

File: conferences.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+    <head>
+        <title>Conferences</title>
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        <h1>Our Conferences</h1>
+        <div style="overflow-x:auto;">
+        <table>
```
Comment (tonygits) [inline conferences.html]: please re-format code?

Comment (tonygits) [inline conferences.html]: please re-format code?

**Comment by tonygits** [comment]
Please request for a review

File: Simian Home Page.html [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
+    <form action="#"> <input type="text" required placeholder="for@example.com"> <button
+            type="button">Submit</button></button></form>
+    </p>
+
+
+</body>
 
-    </body>
-</html>
+</html>
```
Comment (tonygits) [inline Simian Home Page.html:45]: Add extra Line...your cursor should be at L46

File: Simian Home Page.html [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
+        Please <a href="#conferences" .>Click Here</a> to see our range of conferences.
+    </p>
+    <p>Our Management Courses are accredited by <a href="#" target="_blank"> The Drucker Institute</a>
+    </p>
+
+    <p>Please enter your email here to receive our monthly newsletter
+    <form action="#"> <input type="text" required placeholder="for@example.com"> <button
+            type="button">Submit</button></button></form>
+    </p>
+
```
Comment (tonygits) [inline Simian Home Page.html:41]: Remove spaces

File: conferences.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      </tr>
+      <tr>
+        <td>Technology</td>
+        <td>IoT Forum</td>
+        <td><a href="https://www.themunrohotel.co.za/" target="_blank">The Munro Boutique Hotel, Jo'Burg</a></td>
+        <td>30th August 2022</td>
+      </tr>
+    </table>
+  </div>
+</body>
```
Comment (tonygits) [inline conferences.html:42]: Add extra Line...your cursor should be at L42

File: main.css [hunk 5]
```diff
@@ -1,20 +1,56 @@
...
+th {
+    background-color: rgb(7, 7, 121);
+    color: white;
+    padding: 15px;
+
+}
+
+tr:hover {
+    background-color: coral;
+}
```
Comment (tonygits) [inline main.css:67]: Add extra line...your cursor should be at L57

Comment (sivakanili) [reply main.css:67]: Extra line added as advised

**Review by tonygits — APPROVED** [review]

**Pull request merged**